examples/js/web-elements-app/src/app/app.element.ts

```diff
@@ -0,0 +1,400 @@
+/* eslint-disable max-len */
+
+import './app.element.css';
+import { initialize } from '@devcycle/devcycle-js-sdk';
+
+const clientKey = "<YOUR-CLIENT-KEY-HERE>"
```

Comment:
Can we make it match the other example apps with `process.env["NX_CLIENT_KEY"] || "<YOUR-CLIENT-KEY-HERE>"`?